doc/arc/adr_006.md

```diff
@@ -1,21 +1,23 @@
-# TITLE (short noun phrase)
+# Sharding & Routing per Geographic Region
```

Comment:
Let's add the link back to the root readme. See the other ADRs fort examples

Reply:
```suggestion
# [ADR_006](../../README.md) Sharding & Routing per Geographic Region
```